package.json

```diff
@@ -96,6 +97,7 @@
     "eslint-plugin-react": "7.11.1",
     "eslint-plugin-sort-class-members": "1.3.1",
     "eslint-plugin-typescript": "0.12.0",
+    "graphql": "^0.13.0",
```

Comment:
`eslint-plugin-graphql` declares a `peerDependency` on graphql.  Isn't that enough?

Reply:
Makes sense, ty 👍